deploy/packages/bbp-packages.yaml

```diff
@@ -32,6 +32,7 @@ packages:
       - py-basalt@0.2.4
       - py-entity-management@1.0.9
       - py-jupyter-notebook@4.2.3
+      - py-jupyter-notebook@6.0.0
```

Comment:
Is there a reason we need two versions?

Reply:
I remember you were asking to keep at least some, as a general practice. So I kept it :)
I'd be happy to drop `@4` as it makes dependency tree a mess...

Reply:
Ah, that was more for our software. I think for "official" other stuff, we can just move along.

Reply:
Cool, then it's gone ;)

Reply:
Sounds good, looks like dependency hell already.